code/__DEFINES/obj_flags.dm

```diff
@@ -0,0 +1,9 @@
+// Flags for the obj_flags var on /obj
+
+#define EMAGGED 1
+#define IN_USE 2 // If we have a user using us, this will be set on. We will check if the user has stopped using us, and thus stop updating and LAGGING EVERYTHING!
+#define CAN_BE_HIT 4 //can this be bludgeoned by items?
+#define BEING_SHOCKED 8 // Whether this thing is currently (already) being shocked by a tesla
+#define DANGEROUS_POSSESSION 16 //Admin possession yes/no
```

Comment:
how do you plan to lock varedits without locking the entire flag entirely?
(and dont lock it entirely damnit)

Reply:
Why don't you look at /obj/vv_edit_var(vname, vval) in objs.dm to find out